tests/Avalonia.Visuals.UnitTests/Media/SolidColorBrushTests.cs

```diff
@@ -17,5 +17,34 @@ public void Changing_Color_Raises_Invalidated()
 
             Assert.True(raised);
         }
+
+        [Fact]
+        public void Equals_Null_False()
+        {
+            var red = new SolidColorBrush(Colors.Red);
+
+            Assert.False(red.Equals(other: null));
+            Assert.False(red.Equals(obj: null));
+        }
+
+        [Fact]
+        public void Value_Equals_True()
+        {
+            var red1 = new SolidColorBrush(Colors.Red);
+            var red2 = new SolidColorBrush(Colors.Red);
+
+            Assert.True(red1.Equals(red2 as object));
+            Assert.True(red1.Equals(red2 as ISolidColorBrush));
+        }
+
+        [Fact]
+        public void Value_Equals_False()
+        {
+            var red1 = new SolidColorBrush(Colors.Red);
+            var red2 = new SolidColorBrush(Colors.Red, 0.0);
+
+            Assert.False(red1.Equals(red2 as object));
+            Assert.False(red1.Equals(red2 as ISolidColorBrush));
+        }
```

Comment:
I think there should be a test that checks comparing mutable and immutable brushes.

Reply:
I'll add that

Reply:
added

Reply:
I still can't see tests checking equality between `SolidColorBrush` and `ImmutableColorBrush`?